samples/13_alpha_blending.cr

```diff
@@ -9,10 +9,11 @@ renderer = SDL::Renderer.new(window)
 
 background = SDL::IMG.load(File.join(__DIR__, "data", "fadein.png"), renderer)
 modulated = SDL::IMG.load(File.join(__DIR__, "data", "fadeout.png"), renderer)
+modulated.blend_mode = SDL::BlendMode::BLEND
 a = 255
 
 loop do
-  case event = SDL::Event.wait
+  case event = SDL::Event.poll
```

Comment:
No, you want to wait for an event, otherwise we keep redrawing the same image in a tight loop. We'd loop on poll if there was some animation running with FPS limitation. No need to burn the CPU.